README.md

```diff
@@ -39,111 +82,254 @@ brew install node
+### Step 5: Run Locally (w/ Gulp Workflow)
 
-### Run Locally (w/ Gulp Workflow)
-To view the Pattern Library locally (and to watch for any local changes to content/assets), run the `default` gulp task with:
+To view the Pattern Library locally (and to watch for any local changes to
+content/assets), run the `default` gulp task with:
 
 ```
 gulp
 ```
 
-This will run the `default` gulp task defined in `gulpfile.js`, which has the following task dependencies: `['styles', pldoc_styles', 'images', 'pldoc_images', scripts, pldoc_scripts, jekyll-build', 'watch']`
+This will run the `default` gulp task defined in `gulpfile.js`, which has the
+following task dependencies: `pldoc_styles`, `pldoc_images`, `pldoc_scripts`,
+`images`, `watch`.
 
-* The `styles` and `pldoc_styles` tasks compile your css files.
-* `images` and `pldoc_images` copies images from a source folder, performs optimizations, the outputs them into the build folder
+* The `pldoc_styles` task compiles your local PLDOC Sass files into css files.
+* `images` copies images from a source folder, performs optimizations, the outputs them into the build folder
+* `watch` will start watching source files and will re-run the appropriate tasks when those files change. This will also kick off the next two tasks of building/rebuilding the Jekyll site.
 * `jekyll-build` and `jekyll-rebuild` run the local jekyll server/preview and re-process any layouts, posts, or templates changed.
-* `watch` will start watching source files and will re-run the appropriate tasks when those files change.
 
-After running the default task, your local instance of the Pattern Library can be viewable at http://localhost:3000. There are additional views:
+After running the default task, your local instance of the Pattern Library can
+be viewable at http://localhost:3000. There are additional views:
 
 * http://localhost:3000/examples/**/* - where full page examples and demos are kept
 * http://localhost:3000/elements/**/* - permalinks for individually documented pattern library elements
 
-**NOTE: some gulp tasks had to be separated to support both the pattern library 1)elements and 2)reference site UI separately.** Any task prefixed with `pldoc_*` refers to the reference site UI and not the elements.
+## Development - UX Pattern Library Bower Package
+
+If you add or edit any patterns in this repo (found in ``_src/pattern-library/``) or change any bower-centric settings (e.g. ``bower.json``), please do the following:
 
-### Publishing & Viewing Remotely
-The most up-to-date version of the library can be viewed on our public domain, **http://ux.edx.org**
+1. test that the change you've made will not negatively affect the bower package other apps and codebases are relying on
+2. talk to the UX Team about any bumps (based on [semantic versioning](http://semver.org/)) in the bower package's version number that are needed.
```

Comment:
on this note - 1 nice to have might be an email address / other way to reach out in open use cases (or people could use the issues area?)